package.json

```diff
@@ -35,11 +35,13 @@
     "is-my-json-valid": "^2.20.0",
     "json-strify": "^0.1.7",
     "long": "^4.0.0",
+    "moment": "^2.24.0",
     "pre-commit": "^1.2.2",
     "proxyquire": "^2.1.3",
     "semver": "^7.1.0",
     "standard": "^14.0.0",
     "tap": "^12.6.5",
+    "tap-mocha-reporter": "^3.0.9",
```

Comment:
Why was this added?

Reply:
My bad, missed it while merging from upstream.